moveit_plugins/moveit_simple_controller_manager/CMakeLists.txt

```diff
@@ -29,20 +31,28 @@ ament_target_dependencies(${PROJECT_NAME}
   Boost
 )
 
-install(TARGETS ${PROJECT_NAME}
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib
-        RUNTIME DESTINATION bin)
-
+install(
+  TARGETS ${PROJECT_NAME}
+  EXPORT ${PROJECT_NAME}
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  RUNTIME DESTINATION bin
+)
 
 install(DIRECTORY include/ DESTINATION include)
 
 pluginlib_export_plugin_description_file(moveit_core moveit_simple_controller_manager_plugin_description.xml)
+
 ament_export_libraries(${PROJECT_NAME})
+
+ament_export_dependencies(
+  Boost
```

Comment:
I like this grouped export better as well. What do you think about using a list variable for exported dependencies that can be used for `ament_target_dependencies` as well?